egs/fame/s5/local/dict/add_counts.pl

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env perl
```

Comment:
are the files from local/dict used? IIRC (and I might not be right), most of these scripts were written by Dan to serve as a rule-based G2P for English, so I'm not sure how useful they are generally.
Anyway, if you are using it for G2P, perhaps you should use the sequitur scripts?